hugegraph-llm/src/hugegraph_llm/utils/graph_index_utils.py

```
@@ -137,3 +137,44 @@ def import_graph_data(data: str, schema: str) -> Union[str, Dict[str, Any]]:
         # Note: can't use gr.Error here
         gr.Warning(str(e) + " Please check the graph data format/type carefully.")
         return data
+
+def build_schema(input_text, query_example, few_shot):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)